style.css

```diff
@@ -807,523 +28,772 @@
     align-items: center;
     text-align: center;
     text-decoration: none;
-    margin-left: 130px;
+    margin: 20px;
     width: 4.75rem;
     height: 1.25rem;
     color: #6070ff;
   }
 
-  .navlist {
-    display: flex;
-    flex-direction: row;
-    align-items: center;
-    list-style: none;
-    gap: 80px;
-    width: 16.18rem;
-    height: 2.5rem;
-    margin-right: 130px;
+  .nav-bar {
+    margin: 20px;
+    width: 1.1rem;
+    height: 0.75rem;
   }
 
-  .navlink {
-    font-family: "Poppins", sans-serif;
-    font-size: 15px;
-    font-weight: 500;
-    font-style: normal;
-    color: #344563;
+  .intro {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    width: 24.4rem;
+    height: 44.88rem;
+    background-repeat: no-repeat;
+    background-image: url('header-shapesmobile.png');
+    background-color: #fff;
+    border-bottom-left-radius: 60px;
+    position: absolute;
+    top: 47px;
   }
 
-  /* ----------------intro section-------------- */
-
   .greet {
     display: flex;
-    align-items: center;
     font-family: 'Poppins', sans-serif;
     font-style: normal;
     font-weight: 700;
-    font-size: 48px;
-    line-height: 60px;
-    width: 57.5rem;
-    height: 7.6rem;
-    margin-left: 260px;
-    letter-spacing: 0.37px;
-    color: #091e42;
+    font-size: 40px;
+    line-height: 52px;
+    align-items: center;
+    width: 20.44rem;
+    height: 9.37rem;
+    margin: 20px;
+    order: 0;
+    color: #172b4d;
   }
 
   .text {
-    font-family: 'Poppins', sans-serif;
+    width: 20.43rem;
+    height: 9.12rem;
+    font-family: sans-serif, 'Poppins';
     font-style: normal;
     font-weight: 400;
-    font-size: 20px;
-    line-height: 28px;
-    width: 57.5rem;
-    height: 6.1rem;
-    margin-left: 260px;
-    margin-top: 10px;
+    font-size: 16px;
+    line-height: 24px;
+    margin: 0 20px;
+    order: 1;
     color: #344563;
   }
 
   .link {
-    font-family: 'Poppins', sans-serif;
+    font-family: "Poppins", sans-serif;
+    font-size: 15px;
     font-style: normal;
     font-weight: 500;
-    font-size: 16px;
-    line-height: 24px;
-    margin-left: 130px;
-    margin-top: 10px;
+    margin-left: 10px;
+    text-decoration: none;
+    order: 2;
     color: #7f8cff;
   }
 
   .icons {
-    position: absolute;
-    width: 10.64rem;
-    height: 1.25rem;
-    left: 240px;
-    margin-top: 390px;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-evenly;
+    width: 170.17px;
+    flex: none;
+    flex-grow: 0;
+    list-style: none;
+    margin-left: -25px;
+    gap: 8px;
+    order: 3;
```

Comment:
- [ ] **[Optional]** I think it would be better to recheck your previous page design as some of the icons are not visible, and some contents are miss-aligned, Kindly I would be better to check again your design to make your page more professional 👍 👍 

![image](https://user-images.githubusercontent.com/102757126/220102706-5c379a7a-f0f0-4a98-92a0-6ef5903e02da.png)

![image](https://user-images.githubusercontent.com/102757126/220103178-0ee79124-cabc-4ea8-803c-9a1e0f5e8ba6.png)

Reply:
hi, @TuyishimireEric I seem to have opened a pull request on the wrong branch. how can you help me with this?